CHANGES.rst

```diff
@@ -4,6 +4,17 @@ Change Log
 Unreleased
 ----------
 
+**Added**
+
+* Add method :meth:`_fetch_structured_styles` to class :class:`SubredditStylesheet`
```

Comment:
It seems to be PRAW's convention that "private" methods such as `_fetch_structured_styles` and `_upload_style_asset` are not mentioned in the changelog. Could you remove mention of those two methods?

This also causes issues with sphinx (documentation) because private methods aren't included on doc pages. From [the Travis build](https://travis-ci.org/praw-dev/praw/jobs/496031517):
```
Warning, treated as error:
CHANGES.rst:9:py:meth reference target not found: _fetch_structured_styles
The command "sphinx-build -W docs/ /tmp/foo" exited with 2.
```

Reply:
Yup, missed the instructions for running checkstyle pre-PR.  Fixed the style issues and removed the private methods from the CHANGES.rst file